src/mods/depositAll/index.js

```diff
@@ -0,0 +1,35 @@
+import { makeElement } from '../../utils/misc';
+import { getWindow } from '../../utils/game';
+import { deposit } from './helper';
+
+function addDepositAllButton() {
+	const $stash = getWindow('Stash');
```

Comment:
Can this be `getWindow(WindowNames.Stash)`?

Reply:
Updated ui.WindowNames with a stash key and update changes to reference it